packages/presentation/lib/screen/main_my_bloc.dart

```diff
@@ -11,40 +11,41 @@ abstract class MainBloc extends Bloc {
     CheckStateUseCase checkStateUseCase,
     GenerateNumberUseCase generateNumberUseCase,
   ) =>
-      MainBlocImpl(
+      _MainBlocImpl(
         checkStateUseCase,
         generateNumberUseCase,
       );
 
   TextEditingController get editController;
 
-  void generateRandomNumber();
+  void startNewGame();
+
+  void checkNumber();
 }
 
-class MainBlocImpl extends BlocImpl implements MainBloc {
+class _MainBlocImpl extends BlocImpl implements MainBloc {
```

Comment:
_MainBloc seems prettier

Reply:
but then I can't use it in presentation injector. Have to register it there. I think that's okay that abstract class is public